akvo/rest/views/program_results.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+
+from akvo.rest.authentication import TastyTokenAuthentication
+from akvo.rsr.dataclasses import ResultData, IndicatorData, PeriodData, ContributorData
+from akvo.rsr.models import Project, IndicatorPeriod
+from akvo.rsr.models.result.utils import QUANTITATIVE
+from django.shortcuts import get_object_or_404
+from akvo.rest.views.project_overview import _get_indicator_target, is_aggregating_targets
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.decorators import api_view, authentication_classes
+from rest_framework.response import Response
+from rest_framework.status import HTTP_403_FORBIDDEN
+
+
+@api_view(['GET'])
+@authentication_classes([SessionAuthentication, TastyTokenAuthentication])
+def get_program_results(request, program_pk):
+    queryset = Project.objects.prefetch_related('results')
+    program = get_object_or_404(queryset, pk=program_pk)
+    if not request.user.has_perm('rsr.view_project', program):
+        return Response('Request not allowed', status=HTTP_403_FORBIDDEN)
+    results = get_results_framework(program)
+    aggregate_targets = is_aggregating_targets(program)
+    data = {
+        'id': program.id,
+        'title': program.title,
+        'targets_at': program.targets_at,
+        'results': [
+            {
+                'id': result.id,
+                'title': result.title,
+                'type': result.iati_type_name,
+                'indicators': [
+                    {
+                        'id': indicator.id,
+                        'title': indicator.title,
+                        'type': 'quantitative' if indicator.type == QUANTITATIVE else 'qualitative',
+                        'target_value': _get_indicator_target(indicator, program.targets_at, aggregate_targets),
+                        'score_options': indicator.scores,
+                        'cumulative': indicator.cumulative,
```

Comment:
Hi @MichaelAkvo , @zuhdil 
FYI
I've tried to change `indicator.cumulative` to `indicator.is_cumulative()` in order to get the right cumulative indicator status

but I got an error like this 
<img width="701" alt="Screenshot 2023-03-06 at 15 39 49" src="https://user-images.githubusercontent.com/20871229/223938010-5c31c388-c9c5-42fc-8bc5-8bbe047e741e.png">

Reply:
when you guys, have time. could you please help me to solve this issue.
Once it's resolved then this PR is ready for review. Thank you

Reply:
with `indicator.is_cumulative` it works because it uses this dataclass https://github.com/akvo/akvo-rsr/blob/master/akvo/rsr/dataclasses.py#L530

However, the result is still `False`. I have to look into why...